src/View/SSViewer.php

```diff
@@ -474,6 +474,8 @@ protected function includeGeneratedTemplate($cacheFile, $item, $overlay, $underl
         $scope = new SSViewer_DataPresenter($item, $overlay, $underlay, $inheritedScope);
         $val = '';
 
+        // Placeholder for values exposed to $cacheFile
+        [$cache, $scope, $val];
```

Comment:
What difference does this change make? We could also consider the [composer autoloader approach](https://github.com/composer/composer/blob/e42e1156d51ca5494b058b1a7b480bc703c4f57a/src/Composer/Autoload/ClassLoader.php#L437-L445) if isolating this include is a concern for any reason.

Reply:
This is to protect these variables from being accidentally deleted, as the $cacheFile will require these. You'll notice I've taken a similar approach in the installer.

https://github.com/silverstripe/silverstripe-framework/blob/c939737e5c8c484a02a61bcfbc7737f1da48224d/src/Dev/Install/install5.php#L111